include/picongpu/particles/atomicPhysics/stage/RecordChanges.hpp

```diff
@@ -71,13 +71,13 @@ namespace picongpu::particles::atomicPhysics::stage
 
             auto& atomicData = *dc.get<AtomicDataType>(IonSpecies::FrameType::getName() + "_atomicData");
 
-            namespace enums = picongpu::particles::atomicPhysics::enums;
+            namespace s_enums = picongpu::particles::atomicPhysics::enums;
```

Comment:
why do you changed it to `s_enums` and what is the meaning of it?

Reply:
I changed it to avoid shadowing the original namespace and `s_` stands for short(-hand)

Reply:
`s_` == `short` is not obvious but I can live with it.